frontend/smart-kickers-game/src/App.js

```diff
@@ -1,17 +1,29 @@
 import { useEffect, useState } from 'react';
 import './App.css';
-import { resetGame } from './apis/Game';
-import { Button } from './components/Button';
-import GameResults from './components/GameResults.js';
+import { Button } from './components/Button/Button';
+import GameResults from './components/Game/GameResults.js';
+import { resetGame } from './apis/resetGame';
+import GameStatistics from './components/Game/GameStatistics.js';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faPerson } from '@fortawesome/free-solid-svg-icons';
 
 import config from './config';
 
+library.add(faPerson);
```

Comment:
You can create some kind of a config file that does things like this one. Then you can import it in the index.js file, if those things need to be used globally

Reply:
Done